graphql-dgs-codegen-core/src/test/kotlin/com/netflix/graphql/dgs/codegen/CodeGenTest.kt

```diff
@@ -460,6 +459,81 @@ class CodeGenTest {
         assertCompilesJava(dataTypes + interfaces)
     }
 
+    @Test
+    fun generateInterfaceClassWithInterfaceFields() {
+        // schema contains nullable, non-nullable and list types as interface fields  and fields that are
+        // not interfaces
+        val schema = """
```

Comment:
Nit: some of the string templates use margins while others don't. I fine either but imo i think they should be consistent.

Reply:
I agree. I don't want to tackle the other places in this Pr though, since I would like to limit the formatting changes. Will just update for the tests I add.